src/olympia/editors/helpers.py

```diff
@@ -670,6 +729,8 @@ def request_information(self):
         self.log_action(amo.LOG.REQUEST_INFORMATION)
         if self.version:
             self.version.update(has_info_request=True)
+            if not self.addon.is_listed and not self.version.reviewed:
+                self.version.update(reviewed=datetime.datetime.now())
```

Comment:
Why is this updating `reviewed`? There's another update two lines above too. Maybe you can merge them?